Wallapop Architecture/Coordinator.xctemplate/___FILEBASENAME___.swift

```diff
@@ -1,23 +1,23 @@
 import Foundation
 import CommonUI
 
-protocol ___VARIABLE_sceneName___Coordinating: Coordinator {
-}
-
 protocol ___VARIABLE_sceneName___CoordinatorDelegate: AnyObject {
 }
 
-final class ___VARIABLE_sceneName___Coordinator: ___VARIABLE_sceneName___Coordinating {
-    var children = [Coordinator]()
-    var transition: ControllerTransitionStyle?
-    weak var parent: Coordinator?
-    
-    weak var delegate: ___VARIABLE_sceneName___CoordinatorDelegate?
+final class ___VARIABLE_sceneName___Coordinator: Coordinator {
+    private weak var delegate: ___VARIABLE_sceneName___CoordinatorDelegate?
     
-    init() {
+    init(fromViewController: UIViewController, delegate: ___VARIABLE_sceneName___CoordinatorDelegate) {
+        self.delegate = delegate
+        let presentation = ControllerTransitionStyle.Presentation(fromController: fromViewController)
+        super.init(transition: .present(presentation))
     }
     
-    func start() {
+    override func start() {
+        let presenter = ___VARIABLE_sceneName___Presenter(sceneDelegate: self)
```

Comment:
Here we should use Presenter builders, right?
```suggestion
        let presenter = ___VARIABLE_sceneName___PresenterBuilder(sceneDelegate: self)
```